VideoApi/VideoApi.DAL/Commands/AddStaffMemberToConferenceCommand.cs

```diff
@@ -0,0 +1,45 @@
+using System;
```

Comment:
Do we need this command? There is already an AddParticipantsToConferenceCommand that can be used which should work. Looking at the code it is doing almost the exact same thing with the exception that this code doesn't handle linked participants. But otherwise that Command can be used which will end up removing much of the code in this PR?